src/lib/parser.h

```diff
@@ -355,6 +356,7 @@ struct RdbParser {
     int rdbversion;       /* keep aside RDB version */
     uint64_t checksum;
     int debugData;      /* if envvar LIBRDB_DEBUG_DATA=1 then print state machine transitions */
+    char currKeyDbg[20];
```

Comment:
please add a comment.
btw, the size of the buffer 20 (and also 400, and 500) is odd to me.
i'd rather stick with round numbers. :stuck_out_tongue:

Reply:
not that it matters, but i see you changed it to 100, so you probably misunderstood me.
100 is not a round number, 128 is.